config/names.coffee

```diff
@@ -14,7 +14,7 @@ DOMAIN_ENGINE = process.env.DOMAIN_ENGINE || 'balena.io/engine'
 FORUMS_DOMAIN = process.env.DOMAIN_FORUMS || 'https://forums.balena.io'
 BASE_IMAGES_LIB = process.env.BASE_IMAGES_LIB || 'balenalib'
 BASE_IMAGES_CORE = process.env.BASE_IMAGES_CORE || 'balena'
-CLI = process.env.CLI || 'balenaCLI'
+CLI = process.env.CLI || 'balena CLI'
```

Comment:
i'm not sure which we've landed on since https://github.com/balena-io-projects/balena-cli-advanced-masterclass uses the spaced version but https://github.com/balena-io-projects/balena-cli-masterclass does not.

Reply:
I thought I had mistakenly introduced the latter (non-spaced version) as the spaced version seems to be prevalent so I'm going to suggest we settle on the former (for now at least) as the CLI docs themselves use this version. I'll open an issue/PR on CLI masterclass and if worthy of further discussion we can do it there :)

Reply:
PR to update to balena CLI here https://github.com/balena-io-projects/balena-cli-masterclass/pull/17